kvbc/test/replica_state_sync_test.cpp

```diff
@@ -261,7 +261,7 @@ TEST_F(replica_state_sync_on_bft_seq_num_test, cannot_delete_only_block_left_wit
   addBlockWithBftSeqNum(2);  // block 1
   addBlockWithBftSeqNum(3);  // block 2
   // Prune block 1.
-  blockchain_->deleteBlocksUntil(2);
+  blockchain_->deleteBlocksUntil(2, false);
```

Comment:
Isn't it `false` by default?

Reply:
It is an instance of `ReplicaBlockchain`, which does not have a default parameter. I implemented default parameters only for the `KeyValueBlockchain` and below classes.